test/setToken.spec.ts

```diff
@@ -191,24 +184,28 @@ contract("SetToken", (accounts) => {
 
       it("should not work with mismatched quantity of units and tokens", async () => {
```

Comment:
Put the context into a describe block. https://docs.google.com/document/d/1_eBX4js6XkrZdVM3gEsiufYz0cn-Wu4TfAG0Av5bgUw/edit#heading=h.teczyosuxzcd

describe("when there is a mismatch in quantity of units and components"...
beforeEach( () => units.pop)

See other spec files for examples.